src/PowerShellEditorServices/Utility/Extensions.cs

```diff
@@ -135,7 +135,14 @@ public static bool Contains(this IScriptExtent scriptExtent, int line, int colum
 
             if (scriptExtent.StartLineNumber == line)
             {
-                return scriptExtent.StartColumnNumber <= column;
+                if (scriptExtent.StartLineNumber == scriptExtent.EndLineNumber)
+                {
+                    return scriptExtent.StartColumnNumber <= column && scriptExtent.EndColumnNumber >= column;
+                }
+                else
+                {
+                    return scriptExtent.StartColumnNumber <= column;
+                }
             }
 
             if (scriptExtent.EndLineNumber == line)
```

Comment:
I don't remember changing this, and I don't remember using it either. Possibly dead code? We _do_ still have a whole other thing going on with _another_ `IScriptExtent` implementation that isn't the `ScriptRegion` being used by the symbols' logic.

Reply:
https://github.com/fflaten/PowerShellEditorServices/commit/f587eededad7b709d15141aca3f0be7ec4b78473

Not dead. IIRC, that's a fix for a false positive this:
```ps1
function demo { 'hello' }; demo
#                          ^
# Looking for this --------|
```

Reply:
But what uses it? I see no references to this method 👀

Reply:
Oh right 🤦‍♂️ I focused on the diff, but the method itself was only used with FindSymbolVisitor. All replaced by `TryGetSymbolAtPosition(line, column)` 👍

Reply:
Ah, thanks!